docs/infrastructure/index.asciidoc

```diff
@@ -4,83 +4,32 @@
 
 [partintro]
 --
-Use the interactive Infrastructure UI to monitor your infrastructure and
-identify problems in real time. You can explore metrics and logs for common
-servers, containers, and services. 
+The Infrastructure UI enables you to monitor your infrastructure and identify problems in real time.
+You start with a visual summary of your infrastructure where you can view basic metrics for common servers, containers, and services.
+Then you can drill down to view more detailed metrics.
+Alternatively, you can select a single component and view the metrics for that component, or seamlessly switch to view the corresponding logs, application traces or uptime information for that component.
```

Comment:
I'd break long lines like this one on the commas.

Reply:
Also, do we want to capitalize Uptime?

Reply:
According to @gchaps,  capitalise "Uptime" when it's the name of the solution. in this particular instance, I don't think it is being used in that context. I think it's just the name of the thing you are reporting on, as are "logs" and "application traces", but happy to discuss.

By coincidence, I was listening to a WTD podcast this morning and I've found a name for my preferred best practice of splitting lines at sentence-level where the authoring tool supports it (paragraphs otherwise, for the XML-based tools). It's "ventilated prose", and here's why it's recommended practice in Asciidoctor: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line. 

My reasoning has been primarily to make comparisons easy and minimise merge headaches (sentences are likely to be the smallest unit of granularity that you care about when making or merging changes to existing content) and secondly, to make the content easy to read and modify (otherwise you can get a mix of short and long lines, which are harder to read as a logical unit, whereas a sentence is always a logical unit). A third advantage pointed out in the link is that you can easily spot and nuke long sentences. I didn't realise it was a recognised "thing" and had a name! 

So, my preference is to stick with sentences, long or short, but again happy to discuss.

Reply:
"Uptime" vs. "uptime" makes sense to me. Thanks.

As for breaking or not breaking sentences, I'll leave that up to @gchaps since she owns the Kibana book.